### PR TITLE
Adds traffic page

### PR DESCRIPTION
**Pull request by tishan89**
## Purpose
Adds traffic page and fixes some other hyperlink issue. 

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ These are the only two steps that are needed to setup analytics with the new sol
 ### Modern Responsive UIs
 With cloud-based analytics now we have sleek and modern responsive UIs with improved 
  filtering and drill down capabilities. 
-You can access the all new analytics portal through [here](https://analytics.choreo.dev/). Use [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-overview) document as a reference to the composition of each
- dashboard and information that is available through them. [This]({{base_path}}/observe/api-manager-analytics/analytics-usecases/finding-faulty-apis) document will guide you through different usecases
-  you can achieve through analytics portal.
+You can access the all new analytics portal through [here](https://analytics.choreo.dev/). Use
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:24]:
```suggestion
You can access the new analytics portal from [here](https://analytics.choreo.dev/).
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ These are the only two steps that are needed to setup analytics with the new sol
 ### Modern Responsive UIs
 With cloud-based analytics now we have sleek and modern responsive UIs with improved 
  filtering and drill down capabilities. 
-You can access the all new analytics portal through [here](https://analytics.choreo.dev/). Use [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-overview) document as a reference to the composition of each
- dashboard and information that is available through them. [This]({{base_path}}/observe/api-manager-analytics/analytics-usecases/finding-faulty-apis) document will guide you through different usecases
-  you can achieve through analytics portal.
+You can access the all new analytics portal through [here](https://analytics.choreo.dev/). Use 
+[this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction) document as a reference to the
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:25]:
```suggestion
To get an overview of the available dashboards and its capabilities, see the [introduction]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction).
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ These are the only two steps that are needed to setup analytics with the new sol
...
+ composition of each dashboard and information that is available through them.
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:26]:
```suggestion
```

File: en/mkdocs.yml [hunk 4]
```diff
@@ -731,6 +731,7 @@ nav:
                         - Configuring Microgateway: observe/api-manager-analytics/configure-analytics/configure-microgateway.md
                         - Configuring API Gateway: observe/api-manager-analytics/configure-analytics/configure-synapse-gateway.md
                     - Working with Organizations: observe/api-manager-analytics/configure-analytics/working-with-organizations.md
+                    - Working with Alerts: observe/api-manager-analytics/configure-analytics/configure-alerts.md
```
Comment (nilminiwso2) [inline en/mkdocs.yml:734]: Shouldn't we name this page 'Configuring Alerts'?

Comment (tishan89) [reply en/mkdocs.yml:734]: I aligned the name with organizations page. isn't it ok to say working with?

Comment (nilminiwso2) [reply en/mkdocs.yml:734]: 'Working with' is something we would avoid unless we have to...

Isn't it correct to say 'Configuring'?

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md [hunk 5]
```diff
@@ -1,2 +1,57 @@
+# Traffic Page
```
Comment (nilminiwso2) [inline en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md:1]: @tishan89 - I applied a commit directly to this page.
Please check before merging.

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md (hunk 5 above)
Comment (tishan89) [reply en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md:1]: ack

**Pull request merged**